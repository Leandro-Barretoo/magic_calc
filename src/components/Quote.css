.Quote-container {
  margin: 10rem;
  padding: 5rem;
}

.Quote-text {
  font-size: 2rem;
  padding: 5rem;
}
