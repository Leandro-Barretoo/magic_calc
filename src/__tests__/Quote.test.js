import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

describe('Testing the Qoute component', () => {
  it('should render correctly to the dom', () => {
    const quote = renderer.create(<Quote />).toJSON();
    expect(quote).toMatchSnapshot();
  });

  it('should render without errors', () => {
    render(<Quote />);
  });
});