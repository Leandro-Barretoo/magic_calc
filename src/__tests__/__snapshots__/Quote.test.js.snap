// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the Qoute component should render correctly to the dom 1`] = `
<div
  className="Quote-container"
>
  <p
    className="Quote-text"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. -William Paul Thurston
  </p>
</div>
`;
